### ISSUE TITLE
Where is the Max31850K TC support in the v0.5.1 source?

### ISSUE THREAD

**Issue by christaylor99** [UnifiedEngineering/T-962-improvements#100]
I am in the process of building the 4-port UnifiedEngineering thermocouple interface and started looking through the source code for the branch support mentioned in the writeup before installing the new interface board in the T962 oven.
Better thermocouple interface
Werner Johansson  Nov 5, 2015_
https://github.com/UnifiedEngineering/T-962-improvements/wiki/Better-thermocouple-interface
"The MAX31850K utilizes a 1-wire interface as well which makes it a lot easier to interface to the existing board. Master branch code is already capable of detecting and using this interface if present."
Thinking it was in the OneWire code I find only mention of the DS18b20 and DS1822 sensors in the v0.5.1 code.  Is there another prebuilt hex file that contains the Max31850K?

**Comment by christaylor99** [comment]
I looked closer at a smudged copy of the Maxim Max31850 datasheet and saw the family code that looked like "38" and was looking in the onewire code in the src code and saw TC listed as "3b".  Going back "to the well" so-to-speak, I looked at the Maxim site at the datasheet where the family code of 3b is the identity for the MAX31850. So it appears that the 31850 is supported in the v0.5.1 hex as "TC".  Perhaps the next edit should report the chip in the same manner the DS18B20 and DS1822 sensors are currently identified.
